bin/deploy-wmagent.sh

```diff
@@ -191,12 +191,6 @@ unzip -q deployment.zip
 cd deployment-$CMSWEB_TAG
 set +e 
 
-### Patching Couchdb1.6
-echo -e "\n*** Applying deployment patches (couchdb1.6, etc) ***"
-wget -nv https://github.com/dmwm/deployment/pull/162.patch -O - | patch -p 1
-chmod 600 /data/certs/service{cert,key}.pem
```

Comment:
I would keep this line to make sure certs are with the right permissions, otherwise couch1.6 will crash.